### ISSUE TITLE
Support for long-form synthesis.

### ISSUE THREAD

**Issue by computersrmyfriends** [metavoiceio/metavoice-src#120]
The gradio app displays that 
"MetaVoice-1B is a 1.2B parameter base model for TTS (text-to-speech). It has been built with the following priorities:

     **Support for long-form synthesis.

![image](https://github.com/metavoiceio/metavoice-src/assets/323931/1a82e208-e214-43fb-ae22-bc17be3b0a4e)

Is there some way to do it now which is probably not documented?

**Comment by Mohsen1374mt** [comment]
سلام

**Comment by MethanJess** [comment]
@computersrmyfriends 
Long-form support is still in the works, see #53

**Comment by vatsalaggarwal** [comment]
> Is there some way to do it now which is probably not documented?

No, model is built in a way to handle this, but extra code is needed to make it work... we'll release it along with other upcoming features

**Comment by computersrmyfriends** [comment]
For now, is it possible to split text and generate it?

**Comment by computersrmyfriends** [comment]
Still waiting for your long form support. Please let me know if you have any updates

**Comment by lucapericlp** [comment]
Thanks for the enthusiasm & patience @computersrmyfriends, we'll post an update here once we have more 👍

**Comment by fivestones** [comment]
I'm excited for this feature too!

**Comment by fakerybakery** [comment]
Hi, is there any update on when this feature will be released? cc @vatsalaggarwal
